pandas/_libs/tslibs/period.pyx

```diff
@@ -107,7 +107,7 @@ cdef extern from *:
     // must use npy typedef b/c int64_t is aliased in cython-generated c
     static npy_int64 daytime_conversion_factor_matrix[7][7] = {
         {1, 24, 1440, 86400, 86400000, 86400000000, 86400000000000},
-        {0,  1,   60,  3600,  3600000,  3600000000,  3600000000000},
+        {0LL,  1LL,   60LL,  3600LL,  3600000LL,  3600000000LL,  3600000000000LL},
```

Comment:
Which error was this one? Not a blocker but seems strange that only these need the suffix

Reply:
This may be unnecessary? I think I made these changes first, then changed INT32MIN in a second commit. So I'll try with just the change to INT32MIN.

Reply:
Apparently they are needed. When I built with just the change to INT32_MIN, I saw the build warning / error

```
pandas/_libs/tslibs/period.c:674:9: error: this decimal constant is unsigned only in ISO C90 [-Werror]
         {0,  1,   60,  3600,  3600000,  3600000000,  3600000000000},
         ^
```